openeo/rest/connection.py

```diff
@@ -686,7 +686,7 @@ def list_processes(self) -> List[dict]:
         :return: processes_dict: Dict All available processes of the back end.
         """
         data = self.get('/processes').json()["processes"]
-        return VisualList("processes", data = data)
+        return VisualList("processes", data = data, parameters = {'show-graph': True})
```

Comment:
will this trigger the visualization of the process graphs of all process (where available)?

Reply:
Yes, of course only once expanded.